extensions/khr/GLSL_KHR_cooperative_matrix.txt

```diff
@@ -0,0 +1,396 @@
+Name
+GL_KHR_cooperative_matrix
```

Comment:
This file doesn't have the indentation that GLSL specs usually have. Should we fix?